src/utils/maps/getMapStyle.ts

```diff
@@ -0,0 +1,53 @@
+import defaultStyle from '../../../public/data/styles/root.json';
+import openStreetMap from '../../../public/data/styles/OpenStreetMap.json';
+
+export default function getMapStyle(style: any) {
+  return new Promise<any>(async (resolve, reject) => {
+    if (style === 'default') {
+      let result = await fetchTiles(
+        defaultStyle,
+        'https://basemaps.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer'
```

Comment:
Can we be sure this URL does not change. Should be extract it into an environment variable?